src/core/Akka.Tests/Actor/FunctionRefSpecs.cs

```diff
@@ -141,13 +151,11 @@ public async Task FunctionRef_created_by_top_level_actor_must_forward_messages()
         public async Task FunctionRef_created_by_top_level_actor_must_be_watchable()
         {
             var s = SuperActor();
-            var forwarder = GetFunctionRef(s);
```

Comment:
Found the problem - we were calling `GetFunctionRef` which ALSO sends a `GetForwarder` message, thus creating a race condition. Simplified the code and fixed another test case where this could happen too.